rcl/src/rcl/client.c

```diff
@@ -117,6 +117,7 @@ rcl_client_fini(rcl_client_t * client, rcl_node_t * node)
 rcl_client_options_t
 rcl_client_get_default_options()
 {
+  // !!! MAKE SURE THAT CHANGES TO THESE DEFAULTS ARE REFLECTED IN THE HEADER DOC STRING
```

Comment:
Because in this case (as opposed to "every line of code") this is very easy to forget or not even know that the documentation exists elsewhere and the information about what the defaults are used is basically duplicated information, which we can avoid in most code only cases. I'm just trying to help future developers out, you may think it unnecessary, but I think it helps, so unless there is a good reason to exclude it, I will leave it.